src/components/ActionBar/src/ActionBarButton.vue

```diff
@@ -295,25 +305,35 @@ export default {
 }
 
 .MainText {
-	display: flex;
 	gap: 8px;
-	align-items: center;
 	width: max-content;
-	max-width: 100%;
-	overflow: hidden;
-	line-height: 1;
-	white-space: nowrap;
-	text-overflow: ellipsis;
+}
+
+.iconButton .MainText {
+	display: flex;
+}
+
+@media screen and (max-width: 839px) {
+	.MainText {
+		min-width: 50%;
```

Comment:
Can you please add a comment explaining specifically why the MainText needs to be `min-width: 50%` and only on Mobile? It's kinda mysterious otherwise.